charts/templates/statefulset.yaml

```diff
@@ -666,6 +658,10 @@ spec:
         - name: mladmin-secrets
           secret:
             secretName: {{ include "marklogic.authSecretNameToMount" . }}
+        - name: probe-scripts
+          configMap:
+            name: monitor-liveness-probe
```

Comment:
configmap name should be adapted to monitor-liveness-probe-{{ .Release.Name }} see my comment above.